pyproject.toml

```diff
@@ -21,9 +21,9 @@ pysam = "^0.15"
 xphyle = "^4.0.8"
 
 [tool.poetry.dev-dependencies]
-pytest = "^3.0"
+pytest = "^4.0"
 pytest-cov = "^2.7"
-black = {version = "^18.9b0",allows-prereleases = true}
+black = {version = "^19.3b0",allows-prereleases = true}
```

Comment:
If we just always want to be on the latest version of black, you can use 'stable'.